docs/dev_101_services.md

```diff
@@ -5,27 +5,33 @@ sidebar_label: "Custom actions"
 
 Home Assistant provides ready-made actions for a lot of things, but it doesn't always cover everything. Instead of trying to change Home Assistant, it is preferred to add it as a service action under your own integration first. Once we see a pattern in these service actions, we can talk about generalizing them.
 
+[Service actions should always be registered in the integration's `async_setup` or `setup` function](integration-quality-scale/rules/action-setup), not in the integration's `async_setup_entry` or in a platform's `async_setup_entry`, `async_setup_platform` or `setup_platform`.
```

Comment:
Let's add a sentence that explains why we want this. Ie, we want services to always be registered regardless of the state of a config entry or entity. This makes for a better user experience where the user can call the service action and have an informative error message if eg the config entry is not loaded.